x-pack/plugins/spaces/public/management/edit_space/edit_space_features_tab.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
__issue:__ it looks like it's possible to save a solution-specific Space with some of the features toggled off if you do it when in classic mode, but then switch back to solution mode.


https://github.com/user-attachments/assets/754de45f-400e-418b-a042-111c86fa18eb

Reply:
Let's say you are a user that has made a careful selection of visible features for a space that is in Classic view, and saved that. Later, you or someone else edits the space to change it to a solution view - maybe just for exploratory reasons. If you want to change the space back to Classic mode, wouldn't it benefit you to see that your careful selection has been retained? This scenario was demo'd and the current behavior was seen as a good thing.

Storing the disabled features information for a space that is set to a solution view shouldn't be a problem. The API internals shouldn't give complete trust to the information submitted by the client. 

If it's very important for us to remove that information when posting the created / updated space, let me know and I will adjust it.

Reply:
Thanks for explaining this. It was never a big deal, but I'm glad I understand now.

Pushed 9a8e3355272

Reply:
> we're planning to throw an error in API if anyone tries to create or update a Space in "solution" mode with a non-empty disabled features list

True, was added in scope of https://github.com/elastic/kibana/pull/191634/commits/041623b9fec22bcad6f3e8516d743be12cb7be6e.
Thanks for addressing this